rust/tensors.rs

```diff
@@ -6,7 +6,7 @@ use crate::ffi;
 #[derive(Debug, Clone, PartialEq)]
 pub struct TensorDescriptor<'a> {
     pub element_type: ElementType,
-    pub shape: Cow<'a, [ffi::size_t]>,
+    pub shape: Cow<'a, [i32]>,
```

Comment:
Can you think of a good reason to prefer `int` over `size_t` here?

My thinking was that `size_t` is the type you actually use for indexing (e.g. `std::vector<T>::size_type == size_t`) and it is identical to Rust's `usize` type <span title="Well technically usize is identical to untptr_t, but size_t == uintptr_t on pretty much every platform anyway."><sup>1</sup>.</span> 

If we want to stick with `int` we should also probably use [`std::os::raw::c_int`](https://doc.rust-lang.org/std/os/raw/type.c_int.html) instead of `i32`.

Reply:
Originally i exposed size_t  because I thought we were  passing  the data from rust to tf  and we can  check..  but since we are doingin  to_runecoraltensor:
```
    result.shape = tfLiteTensor.dims->data;
```

And since bindgenis treating  int as i32, i had  to   change this..